doto-backend/src/config/passport-setup.js

```diff
@@ -1,6 +1,7 @@
 const passport = require('passport')
 const GoogleStrategy = require('passport-google-oauth20')
 const User = require('../models/User');
+const { logging } = require('../common/logging');
```

Comment:
I think this should be `logger`

Opened a bugfix at #219

Reply:
@Minus20Five, I came across that in my open PR, should be fixed there. For reference #217